checks/python_module_version_test.go

```diff
@@ -0,0 +1,55 @@
+package checks
+
+import (
+	"os/exec"
+	"testing"
+)
+
+func TestPythonModuleVersion(t *testing.T) {
+	// First make sure python is in the PATH
+	// Don't run these tests unless you are on a system with python installed
+	_, err := exec.LookPath("python")
+	if err != nil {
+		return
+	}
+
+	// Test a module exists, don't specify a version
+	err = PythonModuleVersion{Name: "yaml"}.Check()
+	if err != nil {
+		t.Error(err)
+	}
+
+	// A python module you certainly don't want on your system
+	// Don't specify a version
+	err = PythonModuleVersion{Name: "i-am-not-a-python-module-at-all"}.Check()
+	if err == nil {
+		t.Error("Got no error, which is the expected behavior here.", err)
```

Comment:
Can this be reworded to be more clear? 

Expected error but got none
or
Got no error but expected to get one

Reply:
This is going away after #29 is finished since I'm standardizing the check tests. I don't think it's worth blocking this PR on.